tests/testthat/test-5Unifrac.R

```diff
@@ -8,93 +8,74 @@ test_that("Unifrac beta diversity", {
     
     expect_error(
         calculateUnifrac(tse, assay.type = "test", tree_name = "phylo",
-                         weighted = FALSE, normalized = TRUE,
-                         BPPARAM = SerialParam())
+                         weighted = FALSE)
     )
     expect_error(
         calculateUnifrac(tse, assay.type = 2, tree_name = "phylo",
-                         weighted = FALSE, normalized = TRUE,
-                         BPPARAM = SerialParam())
+                         weighted = FALSE)
     )
     expect_error(
         calculateUnifrac(tse, assay.type = TRUE, tree_name = "phylo",
-                         weighted = FALSE, normalized = TRUE,
-                         BPPARAM = SerialParam())
+                         weighted = FALSE)
```

Comment:
These tests are now duplicated

Reply:
Aren't they testing if an error is thrown for different wrong values of `assay.type`?

Reply:
Ahhhh, sorry, my mistake.